presto-kudu/src/main/java/com/facebook/presto/kudu/properties/KuduTableProperties.java

```diff
@@ -321,7 +327,9 @@ public static RangePartition parseRangePartition(String json)
             String partitionRangesValue = mapper.writeValueAsString(rangePartitionList);
             properties.put(RANGE_PARTITIONS, partitionRangesValue);
 
-            // currently no access to numReplicas?
```

Comment:
is comment no longer accurate?

Reply:
Yes, I missed the method for retrieving the numReplicas information.

Reply:
In fact, the method getNumReplicas() was added in version 1.8.0 of the Kudu Client library.